ge/org.osate.ge.fx/src/org/osate/ge/fx/palette/PaletteItem.java

```diff
@@ -0,0 +1,79 @@
+package org.osate.ge.fx.palette;
+
+import java.util.Objects;
+
+import javafx.beans.value.ObservableValue;
+import javafx.beans.value.WeakChangeListener;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Region;
+import javafx.scene.text.Font;
+
+class PaletteItem<I> extends Region {
+
+	private final Button Button;
```

Comment:
Rename all fields and methods which start with a capital letter. All fields should(With the exception of those used as constants) should have a lower camel case name. Review the standard java naming conventions for other similar conventions.

Reply:
I have fixed the naming convention of all fields and methods.